.github/workflows/gatk-tests.yml

```diff
@@ -251,6 +251,19 @@ jobs:
           identifier: Java ${{ matrix.Java }} test on docker ${{matrix.testType}}
           only-artifact: ${{ needs.check-secrets.outputs.google-credentials != 'true' }}
 
+  #test wdl validation
+  wdlValidation:
+    runs-on: ubuntu-latest
+    name: Validate WDLs using womtools
```

Comment:
Can you modify the names (workflow names, name property, gradle task names, and comments) so the "scripts" and "generated" versions are more clearly distinguished from each other (i.e., `validteScriptsWDL` and `validateGeneratedWDL` or something similar). Its hard to tell from looking at just this definition that it's not doing the autogenerated validation.